Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs

```diff
@@ -125,6 +130,10 @@ public MoesifMiddlewareNetCore(RequestDelegate next, Dictionary<string, object>
                     if (isBatchingEnabled)
                     {
                         ScheduleWorker();
+                    } 
+                    else
+                    {
```

Comment:
why is it else, appConfig should always be asynchronous right?

Reply:
We want to start worker to fetch application config only when batching is disabled. In case of batching enable, we'll fetch the application config if needed inside the ScheduleWorker job.

Reply:
there is duplicate code from line 114 to 138. Its trying to do the same thing...reading config first time..can u consolicate the code. sleep after reading once, that way you dont need extra dup code

Reply:
okay, have updated the PR